### ISSUE TITLE
Dependency 

### ISSUE THREAD

**Issue by sneumann** [serbinsh/R-FieldSpectra#23]
Hi,
currently,  the `DESCRIPTION` lists the dependency `PEcAnRTM` under `Suggests`. 
A GitHub action script we are currently creating as PR fails, because it does not know where to get that dependency. This requires a `Remotes: ` information the `DESCRIPTION` that's also part of our PR. 
Yours, Steffen

**Comment by serbinsh** [comment]
Good catch - we can fix this.  A lot of the dev for this happened years ago and so some depends are no longer relevant and/or I didnt do things as well as I should have!
